types/three/examples/jsm/postprocessing/Pass.d.ts

```diff
@@ -12,10 +12,10 @@ export class Pass {
     setSize(width: number, height: number): void;
     render(
         renderer: WebGLRenderer,
-        writeBuffer: WebGLRenderTarget,
-        readBuffer: WebGLRenderTarget,
-        deltaTime: number,
-        maskActive: boolean,
+        writeBuffer: WebGLMultipleRenderTargets | WebGLRenderTarget | null,
+        readBuffer?: WebGLMultipleRenderTargets | WebGLRenderTarget,
```

Comment:
I think the better fix would be to have `WebGLMultipleRenderTargets` extend `WebGLRenderTarget` like it does in the three.js source code.

Reply:
I tried but since the type of texture is an object in `WebGLRenderTarget` and an array in `WebGLMultipleRenderTargets`, its not possible to inherit and change type of the property in typescript. This seemed to be the simplest solution for Passes atleast.
One way is to make `texture` to be `Texture|Texture[]` in `WebGLRenderTarget` but that might break a lot of projects and require special type casting at many places.
Another way is to make an interface like IWebGLRenderTarget and make both implement it. 

I am open to other ideas, would be good to fix type handling of `WebGLMultipleRenderTargets`.

Reply:
Makes sense, I've opened https://github.com/three-types/three-ts-types/pull/519 to resolve this.